### PR TITLE
NOJIRA: Added Swift Package manager support for the core library

### PR DESCRIPTION
**Pull request by me2k**
## JIRA
NOJIRA

## DESCRIPTION
This PR adds support for Swift Package Manager. 
It only supports the Full SuperAwesome package. By adding other sub modules as provided by the Git SubSpecs would require the use of ugly import statements throughout the code.

Support for AdMob can be added, however I feel this add on should be moved to another repository.

## SCREENSHOTS
Screenshot 2021-11-15 at 21 15 38

### CODE REVIEW

File: Pod/Libraries/SUPMoatMobileAppKit.xcframework/ios-arm64/SUPMoatMobileAppKit.framework/Versions/A/Headers/SUPMoatAdEventType.h [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    SUPMoatAdEventComplete
+    , SUPMoatAdEventStart
+    , SUPMoatAdEventFirstQuartile
+    , SUPMoatAdEventMidPoint
+    , SUPMoatAdEventThirdQuartile
+    , SUPMoatAdEventSkipped
+    , SUPMoatAdEventStopped
+    , SUPMoatAdEventPaused
+    , SUPMoatAdEventPlaying
+    , SUPMoatAdEventVolumeChange
```
Comment (markGilchrist) [inline Pod/Libraries/SUPMoatMobileAppKit.xcframework/ios-arm64/SUPMoatMobileAppKit.framework/Versions/A/Headers/SUPMoatAdEventType.h:22]: are these changes part of this pr?

Comment (me2k) [reply Pod/Libraries/SUPMoatMobileAppKit.xcframework/ios-arm64/SUPMoatMobileAppKit.framework/Versions/A/Headers/SUPMoatAdEventType.h:22]: Yes, I noticed that the SUPMoatMobileAppKit.xcframework didn't look like the one I had generated and was missing a Plist, so I swapped it out.  The only thing looking back now is I cannot tell the version of the framework due to that missing Plist.

**Review by dnharjani-sa — APPROVED** [review]

**Comment by gunhansancar** [comment]
Will be considered in the future. Closing for now.

**Comment by markGilchrist** [comment]
can someone with admin rights remove me from the super awesome repos as I no contribute to these repos?

**Comment by dnharjani-sa** [comment]
> can someone with admin rights remove me from the super awesome repos as I no contribute to these repos?

Will do

**Pull request closed without merging**